admin-apis/addApiKeys.js

```diff
@@ -0,0 +1,99 @@
+import { generateKeyPairSync, sign } from "crypto";
+import { generateApiSignature } from "./utils.js";
+import { compressPublicKey } from "./utils.js";
+import { createPrivateKey } from "crypto";
+import axios from "axios";
+import * as dotenv from "dotenv";
+import fetchAppListByAccountKey from "./fetchAllApps.js";
+
+dotenv.config();
+
+const findAppById = async (appId) => {
+  const apps = await fetchAppListByAccountKey();
+  const foundApp = apps.find((app) => app.appId === appId);
+
+  return foundApp;
+};
+
+const updateAppUsingAccountKey = async () => {
+  const timestamp = Date.now().toString();
+  const updatedAppName = "MyApp-Updated";
+
+  // Find the app by ID
+  const app = await findAppById(process.env.APP_ID);
+  if (!app) {
+    throw new Error(`App with ID ${process.env.APP_ID} not found`);
+  }
+  console.log(app);
+
+  // generate a new key pair
+  const keyPair = generateKeyPairSync("ec", {
+    namedCurve: "P-256", // Options
+    publicKeyEncoding: {
+      type: "spki",
+      format: "jwk",
+    },
+    privateKeyEncoding: {
+      type: "pkcs8",
+      format: "jwk",
+    },
+  });
+
+  const publicKeyJwk = keyPair.publicKey;
+  const privateKeyJwk = keyPair.privateKey;
+
+  const publicKeyCompressed = compressPublicKey(
+    Buffer.concat([
+      Buffer.from(publicKeyJwk.x, "base64"),
+      Buffer.from(publicKeyJwk.y, "base64"),
+    ]),
+  );
+
+  const publicKey = publicKeyCompressed.toString("base64");
+
+  const newKey = {
+    name: `Key-${Date.now()}`,
+    api_key: publicKey,
+    signed_hello_message: sign("SHA256", Buffer.from("Hello"), {
```

Comment:
be consistent with casing throughout. Use camelCase

Reply:
done